cluster/plan.go

```diff
@@ -610,13 +610,15 @@ func (c *Cluster) BuildKubeProxyProcess(host *hosts.Host, prefixPath string, svc
 	Binds := []string{
 		fmt.Sprintf("%s:/etc/kubernetes:z", path.Join(prefixPath, "/etc/kubernetes")),
 		"/run:/run",
+		fmt.Sprintf("%s:/lib/modules:z", path.Join(prefixPath, "/lib/modules")),
```

Comment:
Does this need to be read-only? @superseb @leodotcloud 

Asking cuz user mentioned 
```
First approach to solve this would be to add the missing volume of /lib/modules as read-only mount in that function.
```

Reply:
Should only be needed to access (read), not anything else. Readonly should work fine.